### PR TITLE
Adding mankier.com explain() function to explain other commands

### PR DESCRIPTION
**Pull request by trinitronx**
This PR adds an `explain()` function which allows explaining other commands via mankier.com man page data.

Reference: [Explaining Shell Commands in the Shell][1]

[1]: https://www.mankier.com/blog/explaining-shell-commands-in-the-shell.html?hn=1

### CODE REVIEW

File: plugins/available/explain.plugin.bash [hunk 1]
```diff
@@ -0,0 +1,23 @@
+cite about-plugin
+about-plugin 'mankier.com explain function to explain other commands'
+
+# Add this to ~/.bash_profile or ~/.bashrc
```
Comment (nwinkler) [inline plugins/available/explain.plugin.bash]: This comment does not make sense here...

File: plugins/available/explain.plugin.bash [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  elif [ "$#" -eq 1 ]; then
+    curl -Gs "https://www.mankier.com/api/explain/?cols="$(tput cols) --data-urlencode "q=$1"
+  else
+    echo "Usage"
+    echo "explain                  interactive mode."
+    echo "explain 'cmd -o | ...'   one quoted command to explain it."
+  fi
+}
+
+# Update 26-03-2015. If using this command gives no output, see if running a simple fetch causes this error:
```
Comment (nwinkler) [inline plugins/available/explain.plugin.bash]: Do these comments need to be here?

File: plugins/available/explain.plugin.bash [hunk 3]
```diff
@@ -0,0 +1,23 @@
+cite about-plugin
+about-plugin 'mankier.com explain function to explain other commands'
+
+# Add this to ~/.bash_profile or ~/.bashrc
+explain () {
+  if [ "$#" -eq 0 ]; then
```
Comment (nwinkler) [inline plugins/available/explain.plugin.bash]: Can you add the function header/meta function calls (`about`, `group`, `param`, `example`) here?

**Comment by nwinkler** [comment]
I've added a couple of comments - let me know what you think.

**Comment by trinitronx** [comment]
@nwinkler: I've cleaned up comments and added the metadata functions, let me know what you think!

**Comment by nwinkler** [comment]
Thanks!

**Pull request merged**